mewarpx/mewarpx/mwxrun.py

```diff
@@ -201,26 +201,26 @@ def get_gathered_rho_grid(self):
             multifab object is returned on processors other than root.
 
         """
-        return _libwarpx.get_gathered_charge_density_fp(self.lev)
+        return _libwarpx._get_mesh_field_list(_libwarpx.libwarpx.warpx_getGatheredChargeDensityFP, self.lev, None, include_ghosts)
 
-    def get_phi_grid(self):
+    def get_phi_grid(self, include_ghosts=True):
         """Get phi segments on the grid for each tile of each processor.
 
         Returns:
             A list of numpy arrays, the list has an array for every tile and
             each array has dimensions given by the number of cells in that tile.
         """
-        return _libwarpx.get_mesh_phi_fp(self.lev)
+        return _libwarpx._get_mesh_field_list(_libwarpx.libwarpx.warpx_getPhiFP, self.lev, None, include_ghosts)
```

Comment:
Same thing here; I don't think we use the non-gathered grid functions anywhere so if that is the case we should just remove it from mwxrun.py.